src/main/java/htsjdk/variant/variantcontext/writer/VCFWriter.java

```diff
@@ -141,7 +146,7 @@ public void writeHeader(final VCFHeader header) {
     }
 
     public static String getVersionLine() {
-        return VERSION_LINE;
+        return DEFAULT_VERSION_LINE;
     }
 
     public static VCFHeader writeHeader(VCFHeader header,
```

Comment:
There are already a lot of changes in this PR, so I'd prefer to keep this focused for now.
(I like the the general idea of #822, but that one also will need some discussion).